pkg/namespacestore/namespacestore.go

```diff
@@ -321,6 +322,12 @@ func createCommon(cmd *cobra.Command, args []string, storeType nbv1.NSType, popu
 	if len(args) != 1 || args[0] == "" {
 		log.Fatalf(`❌ Missing expected arguments: <namespace-store-name> %s`, cmd.UsageString())
 	}
+	endpoint := util.GetFlagStringOrPrompt(cmd, "endpoint")
+	sigVer, _ := cmd.Flags().GetString("signature-version")
```

Comment:
Does it return v4 in case the user didn't set any signature (the default)? if so, we need to handle this correctly as the user didn't explicitly asked for v4 signature version.

Reply:
one option it mention in error message that default value for `signature-version` is v4 and user has to explicitily select v2 from non-secure endpoint, What do you think?

Reply:
I just think maybe our default should be v2 in this case - and not asking the user to select signature-version when maybe he is not aware of this option.

Reply:
I got this [comment](https://github.com/noobaa/noobaa-operator/pull/1580#discussion_r2031186157) from Liran, In that comment he mentioned that we should not change the value without letting customer know

Reply:
Replied in Liran comment. Resolving this.